.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -35,9 +24,12 @@ Issue Number:
 - [ ] Yes
 - [ ] No
 
-## Are there any new imports or modules? If so, what are they used for and why?
+## New Imports
+Are there any new imports or modules? If so, what are they used for and why?
+
```

Comment:
Do we care about new imports in the examples? We aren't going to be vetting them since these are not released artifacts and this is where we show folks how to use things like Cloud Events that we don't want in our artifacts.

Reply:
will remove for this template